Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1115,7 +1115,7 @@ public static bool SpawnObjects()
             foreach (NetworkIdentity identity in identities)
             {
                 if (ValidateSceneObject(identity))
-                    Spawn(identity.gameObject);
+                    Spawn(identity.gameObject, identity.connectionToClient);
```

Comment:
```suggestion
                    // pass connection so that authority is not lost when server loads a scene
                    // https://github.com/vis2k/Mirror/pull/2987
                    Spawn(identity.gameObject, identity.connectionToClient);
```